### PR TITLE
Adding the version as title to the button

### PR DESCRIPTION
**Pull request by dereuromark**
Adding the version as title to the panel button to quickly know the current running cakephp version, showing

    CakePHP 3.0.6 Debug Kit

on hover now.
Also useful for beginners or lazy ones who quickly want to know the current version the app is running with.

### CODE REVIEW

**Comment by ravage84** [comment]
*Could* be misunderstood for the DebutKit version, though.

**Comment by dereuromark** [comment]
Not really, then it would be `CakePHP Debug Kit 3.x.x`.

**Comment by dereuromark** [comment]
Note the `Welcome to CakePHP v3.0.6 Console` message for CLI :) Which no one would interpret differently, either.

File: src/Template/Requests/view.ctp [hunk 1]
```diff
@@ -22,7 +24,8 @@ use Cake\Routing\Router;
     </li>
     <?php endforeach; ?>
     <li id="panel-button">
-        <?= $this->Html->image('DebugKit.cake.icon.png', ['alt' => 'Debug Kit']) ?>
+        <?= $this->Html->image('DebugKit.cake.icon.png',
+			['alt' => 'Debug Kit', 'title' => 'CakePHP ' . Configure::version() . ' Debug Kit']) ?>
```
Comment (markstory) [inline src/Template/Requests/view.ctp]: The CakePHP version is already show in the Environment panel, why duplicate that here?

Comment (dereuromark) [inline src/Template/Requests/view.ctp]: It is quite hidden there, though, you have to scroll a quite a bit to find it.

**Comment by markstory** [comment]
Ok. Lets give this a shot and see what people think.

**Pull request merged**